src/vs/workbench/contrib/stats/node/workspaceStats.ts

```diff
@@ -502,18 +507,18 @@ export class WorkspaceStats implements IWorkbenchContribution {
 			const packageJsonPromises = getFilePromises('package.json', this.fileService, this.textFileService, content => {
 				try {
 					const packageJsonContents = JSON.parse(content.value);
-					if (packageJsonContents['dependencies']) {
+					if (packageJsonContents['dependencies'] || packageJsonContents['devDependencies']) {
 						for (let module of ModulesToLookFor) {
 							if ('react-native' === module) {
-								if (packageJsonContents['dependencies'][module]) {
+								if (packageJsonContents['dependencies'][module] || packageJsonContents['devDependencies'][module]) {
```

Comment:
dependencies and devDependencies are now possibly undefined

Reply:
I think I addressed this, let me know if there is a better/cleaner way to do the `undefined` check.

Reply:
Looks right now. Did you validate the events already?

Reply:
yep - just need to make sure the GDPR tool gets ran.